.github/workflows/cibuildwheel.yml

```
@@ -101,50 +114,50 @@ jobs:
       id: cache-wheel
       with:
         path: wheelhouse
-        key: wheel-${{ matrix.CIBW_BUILD }}-${{ matrix.CIBW_ARCHS }}-${{ github.sha }}
+        key: wheel-${{ matrix.os }}-${{ matrix.CIBW_BUILD }}
 
     - uses: docker/setup-qemu-action@v3
       if: steps.cache-wheel.outputs.cache-hit != 'true' && runner.os == 'Linux'
 
```

Ok, maybe a separate PR is overkill. This is all you need:

```suggestion
- name: Install OpenSSL (win32)
      if: ${{ endsWith(matrix.CIBW_BUILD, 'win32') }}
      run: |
        Remove-Item -Path "C:\Program Files\OpenSSL" -Force -Recurse  
        vcpkg install openssl:x86-windows
        New-Item -Path "C:\Program Files\OpenSSL" -ItemType SymbolicLink -Value "C:\vcpkg\packages\openssl_x86-windows\"
```

There's a mini build here: https://github.com/qstokkink/libtorrent/actions/runs/11439039359

---

@qstokkink Thank you!, added your change now.

---

@qstokkink Do you think we should use the same vcpkg method for x64? (consistency)

---

In my opinion: absolutely not. This is much slower (it adds 6~7 minutes to the job). That is way too much added time just for the sake of consistency.